boa/benches/exec.rs

```diff
@@ -14,346 +14,5 @@ fn create_realm(c: &mut Criterion) {
-
-fn number_object_access(c: &mut Criterion) {
-    let mut context = Context::new();
-
-    let nodes = Parser::new(NUMBER_OBJECT_ACCESS.as_bytes(), false)
-        .parse_all()
-        .unwrap();
-
-    c.bench_function("Number Object Access (Execution)", move |b| {
-        b.iter(|| black_box(&nodes).run(&mut context).unwrap())
-    });
-}
-
-static BOOLEAN_OBJECT_ACCESS: &str = include_str!("bench_scripts/boolean_object_access.js");
-
-fn boolean_object_access(c: &mut Criterion) {
-    let mut context = Context::new();
-
-    let nodes = Parser::new(BOOLEAN_OBJECT_ACCESS.as_bytes(), false)
-        .parse_all()
-        .unwrap();
-
-    c.bench_function("Boolean Object Access (Execution)", move |b| {
-        b.iter(|| black_box(&nodes).run(&mut context).unwrap())
-    });
-}
-
-static STRING_OBJECT_ACCESS: &str = include_str!("bench_scripts/string_object_access.js");
-
-fn string_object_access(c: &mut Criterion) {
-    let mut context = Context::new();
-
-    let nodes = Parser::new(STRING_OBJECT_ACCESS.as_bytes(), false)
-        .parse_all()
-        .unwrap();
-
-    c.bench_function("String Object Access (Execution)", move |b| {
-        b.iter(|| black_box(&nodes).run(&mut context).unwrap())
-    });
-}
-
-static ARITHMETIC_OPERATIONS: &str = include_str!("bench_scripts/arithmetic_operations.js");
-
-fn arithmetic_operations(c: &mut Criterion) {
-    let mut context = Context::new();
-
-    let nodes = Parser::new(ARITHMETIC_OPERATIONS.as_bytes(), false)
-        .parse_all()
-        .unwrap();
-
-    c.bench_function("Arithmetic operations (Execution)", move |b| {
-        b.iter(|| black_box(&nodes).run(&mut context).unwrap())
-    });
-}
-
-static CLEAN_JS: &str = include_str!("bench_scripts/clean_js.js");
-
-fn clean_js(c: &mut Criterion) {
-    let mut context = Context::new();
-    let nodes = Parser::new(CLEAN_JS.as_bytes(), false).parse_all().unwrap();
-    c.bench_function("Clean js (Execution)", move |b| {
-        b.iter(|| black_box(&nodes).run(&mut context).unwrap())
-    });
-}
-
-static MINI_JS: &str = include_str!("bench_scripts/mini_js.js");
-
-fn mini_js(c: &mut Criterion) {
-    let mut context = Context::new();
-    let nodes = Parser::new(MINI_JS.as_bytes(), false).parse_all().unwrap();
-    c.bench_function("Mini js (Execution)", move |b| {
-        b.iter(|| black_box(&nodes).run(&mut context).unwrap())
-    });
-}
-
-criterion_group!(
-    execution,
-    create_realm,
-    symbol_creation,
-    for_loop_execution,
-    fibonacci,
-    array_access,
-    array_creation,
-    array_pop,
-    object_creation,
-    object_prop_access_const,
-    object_prop_access_dyn,
-    regexp_literal_creation,
-    regexp_creation,
-    regexp_literal,
-    regexp,
-    string_concat,
-    string_compare,
-    string_copy,
-    number_object_access,
-    boolean_object_access,
-    string_object_access,
-    arithmetic_operations,
-    clean_js,
-    mini_js,
-);
+criterion_group!(execution, create_realm);
```

Comment:
As we talked in Discord, we should have an option to keep these benchmarks for easier comparison with previous versions.